py3status/modules/xrandr.py

```diff
@@ -385,6 +385,7 @@ def _apply_workspaces(self, combination, mode):
         """
         if len(combination) > 1 and mode == "extend":
             sleep(3)
+            wm_msg = self.py3.get_wm()["msg"]
```

Comment:
xrandr won't work on Wayland anyway, it's X specific, so I think this module can keep using i3-msg 🙂

Reply:
My understanding is that if you run this on `sway`, you get a single display and `swaymsg`, but it might not or never will pass `if len(combination) > 1 and mode == "extend":` test.

Using `swaymsg` (over `i3-msg`) is still correct even although it might never be used. And I don't want to leave hardcoded `i3-msg` anywhere if this PR is accepted.

Reply:
okay